CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/compose/theme/Dimen.kt

```diff
@@ -0,0 +1,13 @@
+package co.nimblehq.coroutine.ui.screens.compose.theme
+
+import androidx.compose.ui.unit.dp
+
+@Suppress("MagicNumber")
+object Dimen {
+    val Dp0 = 0.dp
+    val Dp4 = 4.dp
+    val Dp8 = 8.dp
+    val Dp16 = 16.dp
+    val Dp24 = 24.dp
+    val Dp52 = 52.dp
```

Comment:
```suggestion
object Dimension {
    val dp0 = 0.dp
    val dp4 = 4.dp
    val dp8 = 8.dp
    val dp16 = 16.dp
    val dp24 = 24.dp
    val dp52 = 52.dp
```

We start the variable with a small alphabet, don't we? Could you please rename it a bit?

Reply:
@lydiasama The reason I use a capital letter first here is that I try to follow the same convention as the variable in `Color `and `Shape` object, as these variables will be used in compose function. What do you think? 🤔

Reply:
I see. It's okay.
Anyway, can you rename the object to `Dimension` instead?

Reply:
@lydiasama Renamed 🙏🏼